express-get/server.ts

```diff
@@ -0,0 +1,61 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+      select *
+      from "films"
+      order by "replacementCost" desc
+      limit 2;
+    `;
+    const result = await db.query(sql);
+    if (!result) res.json([]);
```

Comment:
This check isn't necessary. First, `result` will always be defined so `!result` will always be false. Second, `result.rows` will be `[]` if there are no rows selected by the query, so you can just return `result.rows`.